v2/AdminCore.WebApi.Tests/Controllers/TeamControllerTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using AdminCore.Common.Interfaces;
+using AdminCore.DTOs.Team;
+using AdminCore.WebApi.Controllers;
+using AdminCore.WebApi.Models.Team;
+using AutoFixture;
+using AutoMapper;
+using NSubstitute;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace AdminCore.WebApi.Tests.Controllers
+{
+  public class TeamControllerTests : BaseControllerTest
+  {
+    private readonly ITeamService _teamService;
+    private readonly Fixture _fixture;
+    private readonly IMapper _mapper;
+    private readonly TeamController _teamController;
+
+    public TeamControllerTests()
+    {
+      _teamService = Substitute.For<ITeamService>();
+      _mapper = Substitute.For<IMapper>();
+      _fixture = new Fixture();
+      _teamController = new TeamController(_teamService, _mapper);
+    }
+
+    [Fact]
+    public void CreateTeam_WhenInvalidNamePassed_ReturnsError()
+    {
+      // Arrange
+      var teamToSaveModel = _fixture.Build<CreateTeamViewModel>()
+        .With(x => x.teamName, string.Empty)
+        .Create();
+
+      // Act
+      var result = _teamController.CreateTeam(teamToSaveModel);
+
+      // Assert
+      AssertObjectResultIsNull<TeamDto>(result);
```

Comment:
Base method

Reply:
This is from the BaseControllerTest class